Scripts/Python/grsnPowerOn.py

```diff
@@ -132,7 +132,9 @@
 gearStartingDown = False
 weightEngageDisabled = False
 
-TimerID = xEnum.Enum("UpElevatorSwitch, DownElevatorSwitch")
+class TimerID(enum.IntEnum):
+    UpElevatorSwitch = 1
```

Comment:
I think this will be more Pythonic in cases where we don't need to change the start value:
```suggestion
    UpElevatorSwitch = enum.auto()
```

Reply:
Changed for all of the ones that start at 1!

Would be nice if there was a way to have auto but specify the starting value... Maybe I should take it up to python-dev ;)